doc/SConscript

```diff
@@ -505,7 +505,7 @@ if not skip_doc:
         # XXX Should be in common with reading the same thing in
         # the SConstruct file.
         # bootstrap.py runs outside of SCons, so need to process the path
-        e = Dir(os.path.join('#src', 'engine')).rstr()
+        e = Dir('#').rstr()
         sources = bootstrap.parseManifestLines(e, os.path.join(e, 'MANIFEST.in'))
```

Comment:
I think this is not going to work against the current MANIFEST.in file, but could be wrong.

Reply:
You'll have to tell me... I just dropped src and engine, and after that there was nothing left to "join", there was no other thought behind this.